server/models/functions.sql

```diff
@@ -91,4 +91,53 @@ BEGIN
   );
 END $$
 
+/*
+  PredictAccountTypeId(accountNumber)
+
+  Returns the account type id of the given account number
+*/
+CREATE FUNCTION PredictAccountTypeId(accountNumber INT(11))
```

Comment:
The simple reason is this function is just used when we are in the option 1 and the account number has four digit, and the most important we cannot send this value (the predicted account type id) as parameters because it is not needed at anytime we call the ImportAccount procedure